haskell/src/SwiftNav/SBP/Bootload.hs

```diff
@@ -68,6 +72,8 @@ instance Binary MsgBootloaderHandshakeResp where
     putWord32le msgBootloaderHandshakeRespFlags
     putByteString msgBootloaderHandshakeRespVersion
 
+$(deriveJSON defaultOptions ''MsgBootloaderHandshakeResp)
```

Comment:
Pretty sure it's just the identity (`'id`), but you can pass in some other function. See: https://hackage.haskell.org/package/aeson-0.9.0.1/docs/Data-Aeson-TH.html